.github/actions/run-examples/action.yaml

```diff
@@ -50,7 +50,7 @@ runs:
       with:
         max_attempts: 3
         timeout_minutes: 25
-        command: cd examples/typescript && pnpm install && pnpm test
+        command: cd examples/typescript && pnpm install && pnpm build && pnpm test
```

Comment:
is there a specific reason we need to build it on ci? i think we only should care that nothing breaks in the example logic on ci and not in the package build

Reply:
Yeah, a couple reasons:

1. The `pnpm test` command only checks a subset of the typescript files, not all (e.g., `batch_mint.ts` is not included in `pnpm test`)
2. `tsc` runs full type checking on all `.ts` files in that directory and is typically stricter than `ts-node` which mostly just transpiles and executes the code

the build step runs almost instantly so shouldn't impact CI time at all

Reply:
yea, I am not worried about the time but more on the false failures we can have - anyway lets roll with it and see how it goes

Reply:
Which false failures?

Reply:
I mean, not sure how much we care about type checking, build success, etc when running those examples. We mostly care nothing breaks if we change something in the SDK code

Reply:
> I mean, not sure how much we care about type checking, build success, etc when running those examples

I feel like its important to know if the examples don't compile properly no? Especially if others are trying to use them for their own projects? I think it is achieving the goal you mentioned in the second part of your statement in determining if nothing has broken, especially since it's checking examples that are not included in the `test` script